src/js/components/MultiSelect/MultiSelectContainer.js

```diff
@@ -0,0 +1,640 @@
+                    onClick={(event) => {
+                      if (onChange) {
+                        const nextSelected = options.filter(
+                          (i, index) => !isDisabled(index) || isSelected(index),
+                        );
+                        const nextValue = nextSelected.map((i) =>
+                          valueKey && valueKey.reduce
+                            ? applyKey(i, valueKey)
+                            : i,
+                        );
+
+                        onChange(event, {
+                          option: options,
+                          value: nextValue,
+                          selected: nextSelected,
+                        });
+                      }
+                    }}
+                    onFocus={() => setActiveIndex(-1)}
+                    ref={clearRef}
+                  />
+                )
+              ) : (
+                <Button
+                  a11yTitle={`${value.length} 
+                          options selected. Clear all?`}
+                  label="Clear All"
+                  onClick={(event) => {
+                    if (onChange) {
+                      const nextSelected = options.filter(
+                        (i, index) => isDisabled(index) && isSelected(index),
+                      );
+                      const nextValue = nextSelected.map((i) =>
+                        valueKey && valueKey.reduce ? applyKey(i, valueKey) : i,
+                      );
+                      onChange(event, {
+                        option: options,
+                        value: nextSelected,
+                        selected: nextValue,
+                      });
+                    }
+                  }}
+                  onFocus={() => setActiveIndex(-1)}
+                  ref={clearRef}
+                />
+              ))}
+          </Box>
+        </>
+      ) : (
+        <Text size="small">{`${value.length} selected`}</Text>
+      );
+
+    return (
+      <Keyboard
+        onEnter={onSelectOption}
+        onSpace={onSelectOption}
+        onUp={onPreviousOption}
+        onDown={onNextOption}
+        onKeyDown={onKeyDownOption}
+      >
+        <StyledContainer
+          ref={ref}
+          as={Box}
+          id={id ? `${id}__select-drop` : undefined}
+          dropHeight={dropHeight}
+          a11yTitle="Select dropdown"
+        >
+          {showSelectedInline ? (
+            <Box
+              direction="row"
+              justify="between"
+              flex={false}
+              pad={{ horizontal: 'small', top: 'xsmall' }}
+            >
+              <Box
+                pad={{ top: 'xsmall', bottom: 'xsmall' }}
+                direction="row"
+                justify="between"
+                gap="small"
+                fill="horizontal"
+              >
+                {selectClearAll}
+              </Box>
+              <Button onClick={onClose} a11yTitle="Close Select">
+                <FormUp />
```

Comment:
any reason you didnt do 
`icon={<FormUp />}` within the Button? and take out the `FormUp` as a child?

Reply:
Switched to use `icon={<FormUp />}` in latest push